hash_to_array.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+def hash_to_array(hash)
```

Comment:
Instructions 
> Write a program that collects 5 keys and 5 values from the user, and stores them in a hash

Input needs to be collected from user and this will need to be corrected.

Reply:
Thanks for pointing @josiahandrew12 !

Fixed here https://github.com/aksafan/ROR-lesson-3-and-4/pull/2/commits/ed5e729a522549a8f308e5c9acdbb966cb3cc317